tools/updateIonRangeSlider.R

```diff
@@ -1,35 +1,57 @@
 #!/usr/bin/env Rscript
-
-# This script copies resources from ion.RangeSlider to shiny's inst
-# directory. The ion.rangeSlider/ project directory should be on the same level
-# as the shiny/ project directory.
+# Retrieves a particular version of ion.rangeSlider:
+#  https://github.com/IonDen/ion.rangeSlider
+# Then applies patches from tools/ion.rangeSlider-patches.
 
 # The version of ion.rangeSlider that we use is modified from the base version
 # in two ways:
 # * In our version, mouse events on the slider are not propagated to lower
 #   layers (#711, #1630).
 # * We include a custom skin for Shiny.
+# * When .update() is called, the `.removeProp("tabindex")` is wrapped in
+#   try-catch. This is because this line results in an error with jQuery 3 and
+#   PhantomJS. (#2587)
 
 # This script can be sourced from RStudio, or run with Rscript.
 
-# Returns the file currently being sourced or run with Rscript
-thisFile <- function() {
-  cmdArgs <- commandArgs(trailingOnly = FALSE)
-  needle <- "--file="
-  match <- grep(needle, cmdArgs)
-  if (length(match) > 0) {
-    # Rscript
-    return(normalizePath(sub(needle, "", cmdArgs[match])))
-  } else {
-    # 'source'd via R console
-    return(normalizePath(sys.frames()[[1]]$ofile))
-  }
-}
+# =============================================================================
+# Download library
+# =============================================================================
+# TODO: In the future, use a tag instead of a bare commit hash. See
+# updateBootstrapDatepicker.R for an example.
+version   <- "f0e1116b065c3d091a9c5f04477aaf8894974e2a"
```

Comment:
Minor nitpick: could maybe add some historical context as to why we're using that specific commit?